### PR TITLE
Update version to 5.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hibernate OGM
 
-Version: 5.0.0.Final - 13 May 2016
+Version: 5.1.0-SNAPSHOT
```
Comment (gunnarmorling) [inline readme.md]: We usually only update the date/version during releases. Fine with me, just let's not forget to re-add the date during the next release.

**Comment by gunnarmorling** [comment]
Looks good.

File: readme.md (hunk 1 above)
Comment (Sanne) [inline readme.md]: I don't like that the README on github advertises the final version when it's not.

+1 to remember updating it during release, I always do on Search as one has to set the date.

**Comment by Sanne** [comment]
merged! thanks for the quick check

**Pull request merged**